lib/carbonite/query.ex

```diff
@@ -16,6 +16,7 @@ defmodule Carbonite.Query do
 
   @type prefix_option :: {:carbonite_prefix, prefix()}
   @type preload_option :: {:preload, boolean()}
+  @type order_by_option :: {:order_by, false | nil | tuple()}
```

Comment:
```suggestion
  @type order_by_option :: {:order_by, false | nil | term()}
```

Right? Ecto accepts keyword lists as well as tuples, right? And we don't even need to care, we can accept it as an opaque value in `maybe_order_by`:

```
value -> order_by(queryable, ^value)
```